packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -515,7 +515,7 @@ const ALARM_ACTIONS = [
   },
 ]
 
-const HANDLED_VDI_TYPES = new Set(['system', 'user', 'ephemeral'])
+const HANDLED_VDI_TYPES = new Set(['system', 'user', 'ephemeral', 'pvs_cache'])
```

Comment:
```suggestion
const HANDLED_VDI_TYPES = new Set(['system', 'user', 'ephemeral'])
```

Reply:
Why add the `pvs_cache` type?
`HANDLED_VDI_TYPES` corresponds to the types of `VDI` we want to display.

Reply:
But since `'pvs_cache'` is a `VDI_type`, wouldn't it be better to use that instead of the `name_label` and `name_description` to exclude them? /cc @olivierlambert

Reply:
I don't see this type being used on this forum's message: https://xcp-ng.org/forum/post/81940

Reply:
I was about to post the same answer ^^ There's nothing (for what I can see) in the XAPI record to discriminate it outside the name label & description.

Reply:
I do see a `'pvs_cache'` type here though: https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_vdi.ml#L623-L624

Reply:
Me too, but we need to test how it works in real life.

Reply:
I wouldn't bet about the behaviour on something not even visible in the printed xe record. Even if a name/description isn't perfect, at least we have a proof on how it works, and we have 0 chances to get a PVS server around to test it. So even if it's counter-intuitive, I found the name solution safer than hopping XAPI is doing its work correctly.